extension/fluentbitextension/README.md

```diff
@@ -0,0 +1,56 @@
+# FluentBit Extension
+
+The fluentbit extension facilitates running a FluentBit subprocess of the
+collector.  This is meant to be used in conjunction with the `fluentforward`
+receiver such that the FluentBit subprocess will be configured to send to the
+TCP socket opened by the `fluentforward` receiver.
+
+You are responsible for providing a configuration to FluentBit via the `config`
+config option.  This will be provided to the subprocess, along with a few other
+config options to enhance the integration with the collector.
+
+
+## Example Config
+
+```yaml
+extensions:
+  health_check:
+  fluentbit:
+    executable_path: /usr/src/fluent-bit/build/bin/fluent-bit
+    tcp_endpoint: 127.0.0.1:8006
```

Comment:
Is this the endpoint the output should send to, i.e. our fluentforward receiver's port? Why is this a separate setting and not part of the `config` setting bellow? Is it because we want to generate the OUTPUT section ourselves?

Reply:
Yes, we generate the output config as part of the args to the fluentbit exec call so the user doesn't have to mess with it.

Reply:
The user could optionally override `args` and then set it in the [OUTPUT] section of the config if they wanted.